File: client/directScripts/AWGrun.py
import write
import query
import sys
AWGadd = "TCPIP0::localhost::inst1::INSTR"


def run():
	# AWGrun.run
	Command = ":INIT:IMM"
	return write.write(AWGadd,Command)

	
if __name__ == "__main__":
	if len(sys.argv) == 1:		# run
		result = run()
		print(result)
	else:
		print("err expected 1 or 2 arguments, got "+str(len(sys.argv)-1)+".")
		sys.exit(1)

File: client/directScripts/AWGrun.m
function AWGrun()
% starts signal generation on enabled outputs
% Usage:
%	AWGrun();
% Inputs:
%	none
% Outputs:
%	none


AWGadd = "TCPIP0::localhost::inst1::INSTR";
Command = ":INIT:IMM";

% in the form of ">python (python_command) (device) (device_command)"
cmdStr = "python write.py " + AWGadd + " " + Command;

[status,cmdOut] = system(cmdStr);
if status!=0
    warning("syetem error: "+cmdOut)
end
end